libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -197,8 +201,17 @@ constructor(
                 navigationOptions
             )
         }
+
+        routeRefreshController = RouteRefreshController(tripSession, RouteRefreshApi())
+        routeRefreshController.accessToken = accessToken ?: ""
+        routeRefreshController.intervalSeconds = 10
+        routeRefreshController.refreshRoute { routeRefresh ->
```

Comment:
considering where to move this. May have some iterations. Faster route has a nice entry, where the customer needs to attach a listener to enable

https://github.com/mapbox/mapbox-navigation-android/pull/2577/files#diff-e3ca7b557efaeed833ca6f8f76dc2d03R425